src/misc/setup.cpp

```diff
@@ -45,7 +46,7 @@ extern char** environ;
 using namespace std;
 
 // Commonly accessed global that holds configuration records
-std::unique_ptr<Config> control = {};
+Config* control = {};
```

Comment:
`config`?

Reply:
Done in a rename-separate commit (touched a lot of code)